.travis.yml

```diff
@@ -1,5 +1,6 @@
+sudo: false
 language: ruby
 rvm:
-  - 1.9.2
-  - 1.9.3
-  - 2.0.0
```

Comment:
In all technicality it should.  It's just not tested because it's EOL.  However, there is nothing in the everyday code that is not supported by 1.9+ AFAIW.